### PR TITLE
Document `--plugin-continue-on-error`

### LINKED ISSUE #1720: Document `--plugin-continue-on-error`
Document `--plugin-continue-on-error` CLI option to allow Besu to continue running upon plugin error. See https://github.com/hyperledger/besu/pull/7662 for more info.

### PR DESCRIPTION
**Pull request by alexandratran**
## Description

Document `--plugin-continue-on-error` CLI option.

### Issue(s) fixed

Fixes #1720 

### Preview

https://besu-docs-bbr4ez1nm-hyperledger.vercel.app/public-networks/reference/cli/options#plugin-continue-on-error

### CODE REVIEW

File: docs/public-networks/reference/cli/options.md [hunk 1]
```diff
@@ -2965,6 +2965,45 @@ p2p-port="1789"
...
+```bash
+plugin-continue-on-error=true
+```
+
+</TabItem>
+</Tabs>
+
+Enables or disables continuing to run Besu if a plugin fails during registration or other startup lifecycle stages.
+If set to `true` and any plugin fails, Besu logs the error and continues running.
+If set to `false`and any plugin fails, Besu throws a `RuntimeException` and stops running.
```
Comment (Gabriel-Trintinalia) [inline docs/public-networks/reference/cli/options.md:3003]:
```suggestion
If set to `false` and any plugin fails, Besu throws a `RuntimeException` and stops running.
```

Comment (Gabriel-Trintinalia) [inline docs/public-networks/reference/cli/options.md:3003]:
```suggestion
If set to `false`and any plugin fails, Besu logs the error and stops running.
```

**Review by Gabriel-Trintinalia — APPROVED** [review]
lgtm - some minor text changes

File: docs/public-networks/reference/cli/options.md [hunk 2]
```diff
@@ -2965,6 +2965,45 @@ p2p-port="1789"
...
+
+```bash
+plugin-continue-on-error=true
+```
+
+</TabItem>
+</Tabs>
+
+Enables or disables continuing to run Besu if a plugin fails during registration or other startup lifecycle stages.
+If set to `true` and any plugin fails, Besu logs the error and continues running.
```
Comment (alexandratran) [inline docs/public-networks/reference/cli/options.md:3002]:
```suggestion
If set to `true` and any plugin fails, Besu logs an error and continues running.
```

File: docs/public-networks/reference/cli/options.md [hunk 3]
```diff
@@ -2965,6 +2965,45 @@ p2p-port="1789"
...
+<TabItem value="Configuration file">
+
+```bash
+plugin-continue-on-error=true
+```
+
+</TabItem>
+</Tabs>
+
+Enables or disables continuing to run Besu if a plugin fails during registration or other startup lifecycle stages.
```
Comment (bgravenorst) [inline docs/public-networks/reference/cli/options.md:3001]: Could we perhaps link to the plugin docs?

**Review by bgravenorst — APPROVED** [review]
Minor feedback

**Pull request merged**